xrootd/client/file.go

```diff
@@ -30,6 +35,19 @@ func (f file) Handle() xrdfs.FileHandle {
 	return f.handle
 }
 
+// Close closes the file.
+func (f file) Close(ctx context.Context) error {
+	_, err := f.fs.c.call(ctx, &closereq.Request{Handle: f.handle})
+	return err
+}
+
+// CloseVerify closes the file with verification that the file has provided size.
```

Comment:
`s/with verification that/and checks whether/`
`s/has provided/has the provided/`